src/clarity_vm/tests/forking.rs

```diff
@@ -284,8 +285,10 @@ where
 
     {
         let mut store = marf_kv.begin(&StacksBlockId([0 as u8; 32]), &StacksBlockId([1 as u8; 32]));
-        let mut owned_env =
-            OwnedEnvironment::new(store.as_clarity_db(&TEST_HEADER_DB, &TEST_BURN_STATE_DB));
+        let mut owned_env = OwnedEnvironment::new(
```

Comment:
Can you use `rstest` or some such to make sure we test the forking tests on each epoch, and with all applicable clarity versions?  Thanks!

Reply:
Added in 3a5d8ef83